### ISSUE TITLE
When should CalculationPrice be Close when after-hours activity?

### ISSUE THREAD

**Issue by McQaws** [iexg/IEX-API#222]
## Summary
On Feb 13 for UTX, I see CalculationPrice as PreviousClose as late as 10:20pm with a date of Feb 11 at 7pm.  When should it roll over to showing Close of Feb 13?

## Details
Most of the symbols I tracked Feb 13 showed Close after the market closed.  But, UTX and two others had after hours activity and had SIP data for the CalculationPrice during that hour - fair enough.  The problem is that the SIP price/time showed until 7pm when the quote changed to a type of PreviousClose with a date of Feb 11 at 7pm.  Related, fyi, the /previous endpoint continued to show Feb 12 close data until at least 10:30pm (the time of this posting) for UTX - this is a little later than I'd been seeing the /previous updated.

### Date/time of issue
After close on Feb 13.

### API URL
https://api.iextrading.com/1.0/stock/utx/quote

### What you expected to happen
Well, anything else that would show the Feb 13 day's close after the 5pm after hours session.  As it is, I can't see the Feb 13 close anywhere - I looked at the /ohlc endpoint and it didn't have the close either.

### What actually happened
As described above in details.

**Comment by McQaws** [comment]
Oops - sorry.  In my post I said some of the time stamps were Feb 11 at 7pm - that should have been Feb 12 and midnight.  My timezone logic saw your Feb12 w/midnight time stamp as GMT and I got 5H earlier to my Eastern zone.  The rest stands though - still not sure how to get Close data when there is after-hours quotes.  Any help appreciated.  Nothing I tried worked until closer to 10:40pm Feb 13 (just minutes after my OP) when the /previous endpoint was updated.  Any way to get the close price earlier than that?  Thanks.

**Comment by joshuablackburn-iex** [comment]
Prices during IEX system hours of 8am-5pm ET will be, in order, IEX last trade, if no IEX trade it will be SIP 15 minute delayed price, and if no trades are available the previous close.  Previous close updates overnight between 12-2am.  If you need the current day close, it is available as "close".  The current day close price is updated as we receive them from the listing market.

**Comment by McQaws** [comment]
The sequence you describe is NOT what I observe normally.  I typically see previousclose between 8-8:15a and again between 9:30-9:45a if no trades in that session yet and then the iex/sip as you describe for the remainder of the respective session.  Then, after the 4pm close the calculationPrice changes to CLOSE around 4:15p and has that day's close and keeps that status until the next morning.

But, in this posting I saw sip data for a couple tickers in the after-hours and after 5pm it did not show CLOSE but PREVIOUSCLOSE.  And, the previousclose data was from the day before so I could not see the current day's close.  The quote close and ohlc close were from the day before as well.   

The tickers I track normally don't have any after-hours trades reflected in your /quote.  The only day I saw activity on 3 of the approx 50 tickers the above happened.  I now have a better way to grab all the related data when I see after-hours activity so can report it more completely next time.  

So, to recap, if trade activity is reported in the after-hours, about what time should I expect to see the day's close?  And, where should I best look?  (quote close, ohlc, other?)  Thanks

**Comment by joshuablackburn-iex** [comment]
Closing in favor of #274
